src/main/java/nc/students/ayaz/model/Video.java

```diff
@@ -1,18 +1,27 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
 import java.util.ArrayList;
 import java.util.List;
 
-class Video {
+public class Video {
```

Comment:
Right now i accept String in my ResponseEntity and then call mapper.readValue in which i added my deserializer.
I created my own deserializer because when i accept User (which stores videos as Map) in ResponseEntity, jackson cannot convert it. It says "Sorry, i see a square brackets "[" and "]" inside json string and interprete it as List by default". By the way json string is correct, i checked it through validator.

In future i want to configure restTemplate with my own deserializers